sdk/notificationhubs/notification-hubs/package.json

```diff
@@ -59,34 +58,42 @@
     "url": "https://github.com/Azure/azure-sdk-for-js/issues"
   },
   "homepage": "https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/notificationhubs/notification-hubs/README.md",
+  "//sampleConfiguration": {
+    "productName": "",
+    "productSlugs": [
+      "azure"
```

Comment:
we can add `azure-notification-hubs` product slug

Reply:
Done